raiden/tests/fuzz/test_state_changes.py

```diff
@@ -347,179 +395,337 @@ class InitiatorMixin:
     def __init__(self):
         super().__init__()
         self.used_secrets: Set[Secret] = set()
-        self.processed_secret_requests: Set[SecretRequest] = set()
+        self.processed_secret_request_secrethashes: Set[SecretHash] = set()
         self.initiated: Set[Secret] = set()
 
-    def _get_initiator_client(self, transfer: TransferDescriptionWithSecretState):
-        return self.address_to_client[transfer.initiator]
+    def _available_amount(self, route):
+        client = self.address_to_client[route.initiator]
+        netting_channel = client.address_to_channel[route.hops[1]]
+        return channel.get_distributable(netting_channel.our_state, netting_channel.partner_state)
+
+    def _is_expired(self, secrethash, initiator):
```

Comment:
is `_is_expired` used somewhere else, or why is it outsourced from `_is_removed`?

Reply:
It would be nice to only have `_is_removed` of course, but right now `_is_expired` is still used in two rules in `InitiatorMixin`